interface/patient_file/summary/demographics_full.php

```diff
@@ -619,303 +575,406 @@ function end_group()
+            </div>
+            <div class="col-md-9">
+              <input type='entry' class='form-control form-control-sm mb-1' size='20'
+               name='i<?php echo attr($i); ?>plan_name'
+               value="<?php echo attr($result3["plan_name"] ?? ''); ?>"
+               onchange="capitalizeMe(this);" />
+            </div>
+          </div><!-- end nested row -->
+
+          <div class="form-row"><!-- start nested row -->
+            <div class="col-md-3 pt-1">
+              <span class='label_custom required'><?php echo xlt('Effective Date'); ?>:</span>
+            </div>
+            <div class="col-md-9">
+              <input type='entry' size='16' class='datepicker form-control form-control-sm mb-1'
+               id='i<?php echo attr($i); ?>effective_date'
+               name='i<?php echo attr($i); ?>effective_date'
+               value='<?php echo attr(oeFormatShortDate($result3['date'] ?? '')); ?>' />
+            </div>
+          </div><!-- end nested row -->
+
+          <div class="form-row"><!-- start nested row -->
+            <div class="col-md-3 pt-1">
+              <span class='label_custom required'><?php echo xlt('Policy Number'); ?>:</span>
+            </div>
+            <div class="col-md-9">
+              <input type='entry' class='form-control form-control-sm mb-1' size='16'
+               name='i<?php echo attr($i); ?>policy_number'
+               value="<?php echo attr($result3["policy_number"] ?? ''); ?>"
+               onkeyup='policykeyup(this)' />
+            </div>
+          </div><!-- end nested row -->
+
+          <div class="form-row"><!-- start nested row -->
+            <div class="col-md-3 pt-1">
+              <span class='label_custom required'><?php echo xlt('Group Number'); ?>:</span>
+            </div>
+            <div class="col-md-9">
+              <input type="text" class='form-control form-control-sm mb-1' size='16'
+               name='i<?php echo attr($i); ?>group_number'
+               value="<?php echo attr($result3["group_number"] ?? ''); ?>"
+               onkeyup='policykeyup(this)' />
+            </div>
+          </div><!-- end nested row -->
+
+          <div class="form-row"<?php echo $GLOBALS['omit_employers'] ? " style='display:none'" : ""; ?>><!-- start nested row -->
+            <div class="col-md-3 pt-1">
+              <span class='label_custom required'><?php echo xlt('Subscriber Employer (SE)'); ?>:</span>
+            </div>
+            <div class="col-md-9">
+              <input type='entry' class='form-control form-control-sm' size='25'
+               name='i<?php echo attr($i); ?>subscriber_employer'
+               value="<?php echo attr($result3["subscriber_employer"] ?? ''); ?>"
+               onchange="capitalizeMe(this);" />
+               <span class='small mb-1'><br /><?php echo xlt('if unemployed enter Student'); ?>,
+               <?php echo xlt('PT Student, or leave blank'); ?></span>
```

Comment:
Yes this should be one string, but I decided to leave the original language constants so that existing translations will still work. Don't feel strongly about it though.

Reply:
sounds good to keep as is